Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -606,7 +612,10 @@
   - !type:BuyerJobCondition
     whitelist:
     - Clown
-
+  - !type:BuyerWhitelistCondition
+    blacklist:
+      components:
+      - NukieBundle
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)